python/dask_cudf/record.txt

```diff
@@ -1,65 +0,0 @@
-/conda/envs/gdf/lib/python3.7/site-packages/dask_cudf/core.py
```

Comment:
Can you add this file to `.gitignore` as well?

Reply:
Should already be ignored by the existing rule `python/*/record.txt`. Since the file already slipped into the repo before the rule was put in place I am deleting it here. My assumption is that once deleted, it shouldn't show up in git unless I'm missing something (saw some stuff online about files getting tracked despite being in .gitignore if it was committed before)

Reply:
Resolved by #2851